web/blog/vast-v2.2/index.md

```diff
@@ -96,3 +96,19 @@ summarize:
 In SQL, this would be the `AS` token: `SELECT min(ts) AS min_ts`.
 
 [summarize]: /docs/understand-vast/query-language/operators/summarize
+
+## Regular Expression Predicates
+
+The light at the end of the tunnel for regular expression support is here. VAST
+can now execute query expressions with predicates of the form `x ~ /f.o.*bar/`.
+
+We introduced this functionality "accidentally" during a refactoring that
+transposed predicate evaluation from being row-wise to column-wise. That said,
+the performance is not anywhere near where we would deem it production-grade. We
+have planned a series of optimizations for the next coming releases. Stay tuned!
+
+## Catalog Lookup Tune-Up
+
+As part of our continuous effort to reduce the latency of VAST's catalog, the
+central component that evaluates queries to yield a candidate set of partitions,
+we improved the per-partition lookup time by XXX.
```

Comment:
We should mention that the effect of this is most visible for users spawning many queries in parallel, as the rate at which the catalog does its lookup majorly impacts how well the scheduling algorithm works.

Reply:
Can you take a stab at tweaking the text?

Reply:
We discussed in Slack that we'll be removing the section entirely, as the effect is minimal for most scenarios, and explaining the scenario under which this has a visible effect is too much for this blog post.